web/src/components/PoisDesktop.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { PoiFeature } from 'api-client'
+
+import dimensions from '../constants/dimensions'
+
+const ListViewWrapper = styled.div<{ panelHeights: number }>`
+  min-width: 370px;
```

Comment:
I think 370 is to big. 150 is also enough otherwise if the display is very small, but still counts as a desktop, there is almost no card visiable but only the sidebar.

Reply:
there are always edge cases in responsive design. Here is now a pic of 770px. Since we have no tablet design, we can either start showing desktop mode for large desktops `>992` and use the bottom action bar instead before. but i can not decrease the size for the panel even more

![image](https://user-images.githubusercontent.com/37902063/156561204-019252ee-c119-455d-a1e4-f84f33dc7557.png)

Reply:
Not really a edge case. I just think it would be better / more responsive design if the sidebar width is more responsive. Any argument against changing the min-width to a smaller value?